model/patch/patch.go

```diff
@@ -172,6 +172,9 @@ type Patch struct {
 	// MergeStatus is only used in gitServePatch to send the status of this
 	// patch on the commit queue to the agent
 	MergeStatus string `json:"merge_status"`
+	// MergedFrom is populated with patch id of the existing patch
```

Comment:
nit: "with the"

Reply:
and maybe also an ", if applicable." at the end since this isn't typically populated